src/components/RiddleModal.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+export default function RiddleModalView (props: {riddles: string[]}) {
+  const riddleViews = props.riddles.map(function (val, index) { return <p className='riddle' key={index}>{val}</p> })
```

Comment:
Quick thing to note here! React expects keys to be globally unique, so just using the index as a number probably is going to lead to it yelling at us and maybe causing unexpected behavior.

An easy fix would be to make the key something like `{\`riddle-${index}\`}`, so it's still based on the index but won't clash with anything else

Reply:
Thaaank you, that whole bit caused me quite a bit on confusion because I thought it was the attempt to iterate through the strings that was causing an error regarding it being missing a 'key' property. Let me make that change and push it.